lib/cog_api/http/groups.ex

```diff
@@ -3,17 +3,59 @@ defmodule CogApi.HTTP.Groups do
 
   alias CogApi.Endpoint
   alias CogApi.Resources.Group
+  alias CogApi.Resources.User
 
   def index(%Endpoint{}=endpoint) do
     Base.get(endpoint, "groups") |> Base.format_response("groups", [%Group{}])
   end
 
   def show(%Endpoint{}=endpoint, id) do
-    Base.get(endpoint, "groups/#{id}") |> Base.format_response("group", %Group{})
+    group_json = [
```

Comment:
@jsteiner:

@anellis suggested this solution (and we verified it works):

```elixir

    group_json =
      [get_group(endpoint, id), get_users_for_group(endpoint, id)]
      |> Stream.map(&Task.async(fn -> &1 end))
      |> Enum.map(&Task.await(&1))

    {group_status, group} = group_json |> List.first
    {users_status, users} = group_json |> List.last
```

Any opinions between the two?